pyproject.toml

```
@@ -0,0 +1,52 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+
+requires = [ "setuptools>=75.3.2" ]
+
+[project]
+name = "oauthlib"
+version = "3.3.1"
+description = "A generic, spec-compliant, thorough implementation of the OAuth request-signing logic"
+readme.content-type = "text/x-rst"
+readme.file = "README.rst"
+license = { text = "BSD-3-Clause" }
+maintainers = [ { name = "Jonathan Huot", email = "jonathan.huot@gmail.com" } ]
+authors = [ { name = "The OAuthlib Community" } ]
+requires-python = ">=3.8"
+
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "Operating System :: MacOS",
+  "Operating System :: POSIX",
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: Implementation",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+]
+optional-dependencies.rsa = [ "cryptography>=3" ]
+optional-dependencies.signals = [ "blinker>=1.4" ]
+optional-dependencies.signedtoken = [ "cryptography>=3", "pyjwt>=2,<3" ]
+urls = { Homepage = "https://github.com/oauthlib/oauthlib" }
+
+[dependency-groups]
+test = [ "pytest>=4", "pytest-cov>=2.6", "pytest-subtests" ]
```

It's usually nicer to keep such things multi-line. Version bump diffs will look better.
```suggestion
test = [
  "pytest >= 4",
  "pytest-cov >= 2.6",
  "pytest-subtests",
]
```